manifest-generation/bosh-lite-stubs/experimental/voldriver/drivers.yml

```diff
@@ -0,0 +1,9 @@
+drivers:
```

Comment:
Can you combine this `drivers.templates` into the `volman_overrides` so that there is only one hash that we have to merge? I think it makes sense to have one top level `volman_overrides` hash with both `templates` and `releases` as nested properties.